Better describe the content of various things (#236)
ietf-wg-ohai/oblivious-http: draft-ietf-ohai-ohttp.md

  HPKE KEM ID (16),
  HPKE Public Key (Npk * 8),
  HPKE Symmetric Algorithms Length (16),
  HPKE Symmetric Algorithms (32..524256),
}
~~~
{: #format-key-config title="A Single Key Configuration"}

That is, a key configuration consists of the following fields:

Key Identifier:

: An 8 bit value that identifies the key used by the Oblivious Gateway Resource.

HPKE KEM ID:

: A 16 bit value that identifies the Key Encapsulation Method (KEM) used for the
  identified key as defined in {{Section 7.1 of HPKE}} or [the HPKE KDF IANA
  registry](https://www.iana.org/assignments/hpke/hpke.xhtml#hpke-kem-ids).

HPKE Public Key:

> : The public key used by the gateway, the length of which is determined by the
>   HPKE KEM ID.  The `Npk` parameter is determined by the choice of HPKE KEM,
>   which is defined in {{Section 4 of HPKE}}.

Reviewer: ```suggestion
: The public key used by the gateway. The length of this value depends on the `Npk` parameter,
  which is determined by the choice of HPKE KEM {{Section 4 of HPKE}}.
```

Author: How about:

```suggestion
: The public key used by the gateway. The length of the public key is `Npk`, which is
  determined by the choice of HPKE KEM as defined in {{Section 4 of HPKE}}.
```

Reviewer: Perfect!
